docs/reference-guides/interactivity-api/iapi-about.md

```diff
@@ -0,0 +1,191 @@
+# About the Interactivity API
+
+The Interactivity API is **a standard system of directives, based on declarative code, for adding frontend interactivity to blocks**.
+
+**Directives extend HTML with special attributes** that tell the Interactivity API to attach a specified behavior to a DOM element or even to transform it. For those familiar with [Alpine.js](https://alpinejs.dev/), it’s a similar approach but explicitly designed to work seamlessly with WordPress.
+
+## API Goals 
+
+The main goal of the Interactivity API is to **provide a standard and simple way to handle the frontend interactivity of Gutenberg blocks**.
+
+A standard makes it **easier for developers to create rich, interactive user experiences**, from simple cases like counters or popups to more complex features like instant page navigation, instant search, or carts and checkouts.
+
+All these user experiences are technically possible right now without the Interactivity API. However, the more complex the user experience and the more blocks interact with each other, the harder it becomes for developers to build and maintain sites. There are a lot of challenges they have to figure out themselves. The API aims to provide out-of-the-box means for supporting these kinds of interactions.
+
+To address this challenge the following requirements/goals for the Interactivity API were defined:
+
+- **Block-first and PHP-first**: The API must work well with PHP and the current block system, including dynamic blocks, widely extended in WordPress. It must support server-side rendering. Server-rendered HTML and client-hydrated HTML must be exactly the same. This is important for SEO and the user experience.
+- **Backward compatible**: The API must be compatible with WordPress hooks, which could, for example, modify server-rendered HTML. It must also be compatible with internationalization and existing JS libraries on the site (such as jQuery).
```

Comment:
It is compatible with any existing JS libraries, but it won't work out the box. 

If your block uses jQuery, it won't be compatible with client side navigation. Also, if you add any event with any external library, to any element that its visibility is handled by the Interactivity API, you need to control its behaviour by your self.

Reply:
Compatibility with other blocks is mentioned in [the next line](https://github.com/WordPress/gutenberg/pull/61323/files#diff-fcd8a995f6703c8a130f5507361a3fb7ad50216febc9899718d0fd74c0eb9777R19). Maybe we can just remove that part from here.

Apart from that, I believe that client side navigation probably deserves a separate explanation at some point. It is a complex topic with some nuances.